### PR TITLE
Fix various typos.

### PR DESCRIPTION
**Pull request by roberthoenig**
These typos were found by an experimental upgrade to
eslint-plugin-spellcheck v0.0.9

### CODE REVIEW

File: src/streams/SubscriptionsCard.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ class SubscriptionsCard extends PureComponent<Props> {
 
 export default connect((state: GlobalState, props) => ({
   narrow: props.narrow || [],
-  // Main scrren long longer conatin drawer,
+  // Main screen no longer contains drawer,
```
Comment (borisyankov) [inline src/streams/SubscriptionsCard.js]: I am not sure if that comment makes sense as part of the source code.
Maybe it did make sense as a commit message. I would prefer to completely remove it.

Comment (roberthoenig) [reply src/streams/SubscriptionsCard.js]: Sure. Should be done in a follow-up commit?

Comment (borisyankov) [reply src/streams/SubscriptionsCard.js]: Sure, a spellchecking vs removing a comment sounds like a different type of work.

**Comment by gnprice** [comment]
Merged -- thanks @roberthoenig and reviewers!

**Pull request merged**